CHANGELOG.md

```diff
@@ -5,6 +5,13 @@
   list.
   ([#69](https://github.com/dart-lang/yaml_edit/issues/69))
 
+- Fix error thrown when inserting in nested list using `spliceList` method
+  ([#83](https://github.com/dart-lang/yaml_edit/issues/83))
```

Comment:
Feel free to file this as a separate PR, if you want. Then we get to land it quickly :D

Tagging it along here is harmless though.